Userland/ls.cpp

```diff
@@ -156,7 +160,10 @@ bool print_filesystem_object(const char* path, const char* name) {
     else
         printf("%c", st.st_mode & S_IXOTH ? 'x' : '-');
 
-    printf(" %4u %4u", st.st_uid, st.st_gid);
+//    printf(" %4u %4u", st.st_uid, st.st_gid);
+    pw = getpwuid(st.st_uid);
+    gr = getgrgid(st.st_gid);
+    printf(" %s %s ", pw->pw_name,gr->gr_name);
```

Comment:
We should use field width specifiers here to make sure the columns are aligned even if there are usernames with different length.